src/main/java/com/linkedin/kafka/clients/utils/DefaultHeaderSerde.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+package com.linkedin.kafka.clients.utils;
+
+import java.nio.ByteBuffer;
+
+
+/**
+ * Utilities to implement header serializer and deserializer.
+ */
+public interface DefaultHeaderSerde {
```

Comment:
Can we either move some (probably all) of the things in this class to HeaderUtils and rename this class to something like HeaderConstants?